website/src/pages/Blog/styles.ts

```diff
@@ -45,17 +45,16 @@ const useStyles = makeStyles((theme: Theme) => ({
     margin: theme.spacing(3, 0, 2.5),
     [theme.breakpoints.down("xs")]: {
       flexDirection: 'column',
-      margin: theme.spacing(0),
-      gap: theme.spacing(1)
+      margin: theme.spacing(0)
     },
   },
   large: {
     width: '60px',
     height: '60px',
     marginRight: theme.spacing(1.8),
     [theme.breakpoints.down("xs")]: {
-      width: '32px',
-      height: '32px',
+      width: '48px',
+      height: '48px',
```

Comment:
Why this change?

Reply:
It should be `32px` for mobile devices. https://www.figma.com/file/PkV24kmePqdjQiTBJMcJ1L/OpenEBS-landing-page-final?node-id=4%3A10250

Reply:
ohh ok, I thought it's `32px` is by mistake. Because both `top author icon` as well as `author icon on the individual card ` was same.